controllers/etcd_controller.go

```diff
@@ -554,6 +561,12 @@ func (r *EtcdReconciler) getConfigMapFromEtcd(etcd *druidv1alpha1.Etcd, rendered
 	if err = decoder.Decode(&decoded); err != nil {
 		return nil, err
 	}
+
+	decoded.Name = fmt.Sprintf("etcd-bootstrap-%s", string(etcd.UID[:6]))
+	config := decoded.Data["etcd.conf.yaml"]
+	config = fmt.Sprintf("%s\ninitial-cluster: %s", config, values["initialCluster"])
+
+	decoded.Data["etcd.conf.yaml"] = config
```

Comment:
Here we are modifying the rendered template. Why not make the change in the configmap template directly? Secondary changes after rendering will get quickly become confusing.

Reply:
Okay. You mean in configmap of chart directory?